packages/common/scss/leaders.scss

```diff
@@ -0,0 +1,60 @@
+.leaders {
+  &__items {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    padding-left: 0;
+    list-style-type: none;
+    @include media-breakpoint-down($theme-responsive-text-breakpoint) {
+      display: initial;
+      width: 100%;
+    }
+  }
+
+  &__item {
+    .btn-link:hover,
+    .btn-link:active,
+    .btn-link:focus {
+      text-decoration: none;
+    }
+  }
+}
+
+.node-list {
+  $self: &;
+  &--centered {
```

Comment:
Make this an element modifier, e.g. `.node-list__header--centered`
Remove __node and __footer